data/project/emberData/release.md

```diff
@@ -4,12 +4,12 @@ baseFileName: ember-data
 filter:
   - /ember-data\./
 repo: emberjs/data
-initialVersion: 3.15.0
-initialReleaseDate: 2019-12-18
-lastRelease: 3.15.0
-futureVersion: 3.16.0
+initialVersion: 3.16.0
+initialReleaseDate: 2020-02-12
```

Comment:
According to https://libraries.io/npm/ember-data the release date was January 25th

```suggestion
initialReleaseDate: 2020-01-25
```